compiler/src/iree/compiler/Codegen/Common/MaterializeUserConfigs.cpp

```diff
@@ -34,6 +34,14 @@ llvm::cl::opt<std::string> clCodegenTransformDialectStrategyName(
         "library of transform specs (@library_call)"),
     llvm::cl::init(""));
 
+llvm::cl::opt<std::string> clCodegenTransformDialectConfigurationName(
```

Comment:
Whats the difference between a strategy and a configuration.

Reply:
Had an offline chat about this. Maybe you can just use the flag above to combine the two . Say `library::sequence` ? and if `sequence` is missing you use `__transform_main`. 

(Side note :If I understand correctly what this is carrying is the symbol name for the sequence. If so maybe rename from "configuration" to "sequence")

Reply:
Resolving as duplicate with the below.